### PR TITLE
Fixed Bluetooth and added gst-plugins-bad

### PR DESCRIPTION
**Pull request by PetervdPerk-NXP**
Adds gst-plugins-bad packages to the image.
Added hciattach.service to enable bluetooth.

### CODE REVIEW

File: meta-hovergames-distro/recipes-hovergames/images/imx-image-hovergames-demo.bb [hunk 1]
```diff
@@ -399,6 +541,30 @@ fakeroot do_fix_ldconfig() {
...
+    echo >> "${APTGET_CHROOT_DIR}/lib/systemd/system/hciattach.service" "[Unit] \n
+Description=Configure Bluetooth Modems connected by UART \n
+Before=bluetooth.service \n
+After=dev-serial1.device \n
+\n
+[Service] \n
+Type=forking \n
+ExecStartPre=-/usr/bin/hciattach /dev/ttymxc0 qualcomm -t120 115200 flow \n
+ExecStart=-/usr/bin/hciattach /dev/ttymxc0 qualcomm -t120 115200 flow \n
+ExecStopPost=-/usr/bin/hciattach /dev/ttymxc0 qualcomm -t120 115200 flow \n
```
Comment (thochstein) [inline meta-hovergames-distro/recipes-hovergames/images/imx-image-hovergames-demo.bb:557]: What's going on here? You issue the same command twice on startup? You issue the start command on stop?

File: meta-hovergames-distro/recipes-gnome/gdk-pixbuf/gdk-pixbuf_2.38.2.bbappend [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#
+# This class will generate the proper postinst/postrm scriptlets for pixbuf
+# packages.
+#
+
+
+pixbufcache_common() {
```
Comment (thochstein) [inline meta-hovergames-distro/recipes-gnome/gdk-pixbuf/gdk-pixbuf_2.38.2.bbappend:7]: What's going on in these bbappends? Ubuntu rootfs magic I guess. If you learned something you might comment the files or the commit.

Comment (PetervdPerk-NXP) [reply meta-hovergames-distro/recipes-gnome/gdk-pixbuf/gdk-pixbuf_2.38.2.bbappend:7]: So this recipe originally inherits the pixbuf bbclass which causes the the post-install errors when making the demo image. Yocto doesn't have a way to remove an inherit but you can declare them as an empty function which will override the original inherited function.

File: meta-hovergames-distro/recipes-hovergames/images/imx-image-hovergames-demo.bb (hunk 1 above)
Comment (PetervdPerk-NXP) [reply meta-hovergames-distro/recipes-hovergames/images/imx-image-hovergames-demo.bb:557]: Indeed it looks a bit odd and thus probably wrong, it should be only started once, funny enough I don't see multiple instances of hicattach when running this service on the navq.

**Pull request merged**